controllers/kmsissuer_controller.go

```diff
@@ -98,7 +104,72 @@ func (r *KMSIssuerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 			return ctrl.Result{}, r.manageFailure(ctx, log, issuer, err, "Failed to update the issuer with the issued Certificate")
 		}
 	}
-	return ctrl.Result{}, r.manageSuccess(ctx, log, issuer)
+	return ctrl.Result{
+		RequeueAfter: time.Until(desiredCert.NotAfter.Add(-1 * issuer.Spec.RenewBefore.Duration)),
+	}, r.manageSuccess(ctx, log, issuer)
+}
+
+// setIssuerDefaultValues
+func setIssuerDefaultValues(issuer *kmsiapi.KMSIssuer) {
+	if issuer.Spec.Duration == nil || issuer.Spec.Duration.Duration == 0 {
+		issuer.Spec.Duration = &metav1.Duration{Duration: DefaultCertDuration}
+	}
+	if issuer.Spec.RenewBefore == nil {
```

Comment:
How can someone disable automatic renewal? If someone doesn't want to renew a certificate.

Reply:
I am trying to follow the same convention that cert-manager follows to keep the UX consistent
https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1.CertificateSpec

Reply:
ok, I see `cert-manager` forces renewal too https://github.com/jetstack/cert-manager/blob/1bfec374825bee7557ad915e1f015fa7f56ef4a7/pkg/apis/certmanager/v1alpha2/const.go#L31-L32